pages/carpentries.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page-fullwidth
+title: "The Carpentries"
+permalink: /carpentries/
+---
+
+HPC Carpentry aspires to become an official Carpentries program...but we are not there yet! Meanwhile, we link
```

Comment:
Please limit lines to 99 characters or fewer.

Reply:
Did my best!